### DISCUSSION TITLE
move chia full node from linux to  windows

### DISCUSSION THREAD

**Discussion by heeroz** [Chia-Network/chia-blockchain#4276] [Q&A]
My linux chia full node has some problems in remote desktop such as xrdp and VNC. So I want to move chia full node from linux to  windows. Is there any reference I can refer to. Thank you.

**Comment by piotrkochan** [comment]
Just use your mnemonic passphrase and then select paths where plots are available.

**Reply by heeroz** [reply]
Thank you. I will try this.
